blocks/author-profile/author-profile.js

```diff
@@ -0,0 +1,71 @@
+import {
+  createOptimizedPicture,
+  getMetadata,
+} from '../../scripts/aem.js';
+import {
+  div, h2, p, a, span, img,
+} from '../../scripts/dom-builder.js';
+import ffetch from '../../scripts/ffetch.js';
+
+const AUTHOR_INDEX = '/authors-index.json';
+const fallbackPath = '/author/name';
+const fallbackName = 'Name';
+const fallbackDescription = 'Author';
+const fallbackLinkText = 'See more by this author';
+const fallbackLinkAriaLabel = 'Read more';
+
+const breakpoints = [
+  { media: '(min-width: 980px)', width: '2000' },
+  { width: '750' },
```

Comment:
I think this is way to large for the small headshot thumbnail. Why do you need 2000px with image on desktop?

Reply:
Replaced with 120px for all breakpoints